.gitignore

```diff
@@ -3,6 +3,13 @@
 __pycache__/
 *.py[cod]
 *$py.class
-
+*~
+\#*\#
+/.emacs.desktop
```

Comment:
It's good practice to put IDE-specific rules on your machine's global `.gitignore` file instead of the project-level one.

If you still feel this belongs here, can you add a comment saying this section is for emacs?